ns1/ipam.py

```diff
@@ -1050,12 +1053,45 @@ def update(self, address_family, options, server_options=None):
             self.server_options = server_options
 
         self.address_family = address_family
-        self.__dict__.update(options)
+        self.options = options
         self.__dict__.update(server_options)
+
         self.option_list = {
             "options": [
-                {"name": "%s/%s" % (self.address_family, key), "value": value}
-                for key, value in options.items()
+                option.generate_option(address_family)
+                for option in self.options
             ]
         }
+
         self.option_list.update(self.server_options)
+
+
+class DHCPOptionValue:
+    def __init__(self, key, value, always_send=None):
+        """
+        Create the DHCPOptionValue class that can be used as value with :class:`ns1.ipam.DHCPOptions`
+
+        :param key str option name
+        :param value any option value
+        :param always_send bool indicates whether this option be sent back in lease or not
+        """
+        self.key = key
+        self.value = value
+        self.always_send = (
+            bool(always_send) if always_send is not None else always_send
+        )
+
+    def generate_option(self, address_family):
+        """
+        Generates dhcp option value with a proper format
+
+        :param str address_family one of dhcpv4 or dhcpv6 family name
+        """
+        option = {
+            "name": "%s/%s" % (address_family, self.key),
+            "value": self.value,
+        }
+        if self.always_send is not None:
+            option.update({"always_send": self.always_send})
```

Comment:
Suggestion:
```
if self.always_send is not None:
    option["always_send"] = self.always_send
```

Reply:
Your suggestion works two times faster (0.7 against 0.3 ms), but I think update via object is more readable, but it's my opinion. I think we don't need here such a super-fast reaction.

Reply:
It’s not really about speed. It’s common in python to assign values to single keys in this way, so it is perfectly readable, if not idiomatic. Update is used more commonly to set or change multiple keys at one time.

Reply:
Driveby plus one for idiom -- agree it'd standard to use __setitem__ syntax here, i.e. bracket notation.

Reply:
Thank you! I got your point
Will fix this

Reply:
fixed